component/admin/controllers/wrappers.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * @package     RedSHOP.Backend
+ * @subpackage  Controller
+ *
+ * @copyright   Copyright (C) 2008 - 2020 redCOMPONENT.com. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE
+ */
+
+defined('_JEXEC') or die;
+
+/**
+ * Wrappers controller
+ *
+ * @package     RedSHOP.backend
+ * @subpackage  Controller
+ * @since       __DEPLPOY_VERSION__
+ */
+class RedshopControllerWrappers extends RedshopControllerAdmin
+{
+	/**
+	 * Method to favour a list of rating items
+	 *
+	 * @return  void
+	 *
+	 * @since   __DEPLPOY_VERSION__
+	 */
+	public function FVpublish()
```

Comment:
@hoang-tran-redweb i think this function is redundancy when refactor it will use function published of parent class